bot/skills/kozula.py

```diff
@@ -0,0 +1,47 @@
+import logging
+import xml.etree.ElementTree as ET
+import requests
+from mode import cleanup_queue_update
+
+from telegram import Update
+from telegram.ext import Updater, CommandHandler, CallbackContext
+
+logger = logging.getLogger(__name__)
+
+KOZULA_RATE = 600000
+CBR_URL = "https://cbr.ru/scripts/XML_daily.asp"
+
+
+def add_kozula(upd: Updater, handlers_group: int):
+    logger.info("registering tree handlers")
+    dp = upd.dispatcher
+    dp.add_handler(CommandHandler("kozula", kozula, run_async=True), handlers_group)
+
+
+def kozula(update: Update, context: CallbackContext):
+    request = requests.get(CBR_URL)
+    root = ET.fromstring(request.content)
+    for child in root:
+        if child.attrib["ID"] == "R01235":
+            usd_rate = float(child.find("Value").text.replace(",", "."))
+
+    logger.info("USD rate: %s", usd_rate)
```

Comment:
Looks like the variable 'usd_rate' might be referenced before the assignment. What if we do not get 'usd_rate' for some reason (network spike for example)? 

Maybe, we should care about this kind of case. 
Shall we?

Reply:
Yes, we should.
I`ve merged online by misclick, but It seems ok?

Reply:
Oh, I though you fixed `black` error about quotes in it's settings? But It still decline. It's too important to do it manually and always in the same way? Position of import module must be in right place too? :) Looks funny.